lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -336,6 +333,20 @@ public void TestDeleteDirectorySecondLevelE2E()
             Assert.True(CheckContainerExists(_container));
         }
 
+        private List<CloudBlockBlob> CreateTempBlobs(string directory, int fileCount = 3)
+        {
+            var blockBlobs = new List<CloudBlockBlob>();
+            for (var i = 0; i < fileCount; i++)
+            {
+                var filePath = directory + '/' + i;
+                var blockBlob = _container.GetBlockBlobReference(filePath);
+                UploadFromString(blockBlob, "hello");
+                Assert.True(CheckBlobExists(blockBlob));
+                blockBlobs.Add(blockBlob);
+            }
+            return blockBlobs;
+        }
```

Comment:
Maybe use Linq? e.g.
```C#
private IEnumerable<CloudBlockBlob> CreateTempBlobs(string directory, int fileCount = 3)
{
    return Enumerable.Range(0, fileCount).Select(i =>
    {
        var filePath = directory + '/' + i;
        var blockBlob = _container.GetBlockBlobReference(filePath);
        UploadFromString(blockBlob, "hello");
        Assert.True(CheckBlobExists(blockBlob), "Blob does not exist: " + filePath);
        return blockBlob;
    });
}
```

Reply:
changed

---
In reply to: [198681302](https://github.com/apache/reef/pull/1473#discussion_r198681302) [](ancestors = 198681302)